src/jaxsim/api/model.py

```diff
@@ -260,7 +279,7 @@ def frame_names(self) -> tuple[str, ...]:
             The names of the links in the model.
         """
 
-        return tuple([frame.name for frame in self.description.get().frames])
+        return tuple([frame.name for frame in self.description.frames])
```

Comment:
We could use a generator here
```suggestion
        return tuple(frame.name for frame in self.description.frames)
```